vips.go

```diff
@@ -312,6 +319,8 @@ func (img *vipsImage) Rotate(angle int) error {
 		return vipsError()
 	}
 
+	C.vips_autorot_remove_angle(tmp)
```

Comment:
Is this the right place to strip orientation tags? I also thought about doing it when fetching the image orientation during extractMeta(), but then a second call to extractMeta() would return different values.

Another place where I think this would make sense is vipsImage.Save().